docs/curation-review-tut.md

```diff
@@ -0,0 +1,417 @@
+- [schema guide](https://github.com/chanzuckerberg/single-cell-curation/blob/main/docs/schema_guide.md)
+
+<details>
+<summary> Table of contents </summary>
+
+- [Quick start](#quick-start)
+- [Why contribute?](#why-contribute)
+- [Basic data requirements](#basic-data-requirements)
+- [Anndata object requirements](#anndata-object-requirements)
+- [Schema definition](#schema-definition)
+- [Cellxgene curation tools](#cellxgene-curation-tools)
+
+</details>
+
+<br/>
+
+---
+
+## Quick start
+
+<br/>
+
+If you already know everything you need to know, then give curation a shot with this [quick start guide](https://github.com/chanzuckerberg/single-cell-curation#quick-start). Otherwise, continue reading!
+
+---
+
+## Why contribute?
+
+<br/>
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111403110-d1589700-86a2-11eb-99dd-bf52e348d43a.png) --->
+<br/>
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/111403110-d1589700-86a2-11eb-99dd-bf52e348d43a.png" width="1000">
+</p>
+
+<p align="center">
+  <b> Figure: </b> Collection View of the Cellxgene Data Portal
+</p>
+
+<br/>
+
+Reason to contribute:
+ - link permanence, allows you to reference data portal links in publication with guarantee that the link will always be there
+ - sharing of private datasets with collaborators
+ - easy for readers to explore your dataset (with no extra work on your side - goodbye shiny!)
+ - accesibility of your dataset through many of the popular single cell toolchains (link to an instance of the cellxgene explorer app)
+
+<br/>
+ 
+## Basic data requirements
+
+<br/>
+
+The following elements of your single cell dataset are required. One of the prerequisites for contributing data to the portal is that you have these components represented as an `anndata` object (more on that in the next section). Note that, in addition to the following, other representations of the expression matrix (alternative normalization, SCTransform, corrected counts from SCTransform or background correction) can all be stored as `layers` your anndata object.
+
+- count matrix (except for certain assays)
+- normalized expression matrix used for visulization
+- cell level metadata
+- variable/feature level metadata (not required)
+- embedding (at least one required, UMAP, tSNE, spatial, PCA)
+
+
+
+<br/>
+
+
+## Anndata object requirements
+
+<br/>
+
+Contributing data to the cellxgene data portal or visualizing your data with the cellxgene explorer requires that your data is represented as an anndata data object. You can read more about Anndata through its [documentation site](https://anndata.readthedocs.io/en/latest/). Additionally, you can check out some of the requirements for using an anndata object with cellxgene [here](https://chanzuckerberg.github.io/cellxgene/posts/prepare). In this section of the tutorial, we will augment some of this information with some considerations that may be specific to the cellxgene curation process.
```

Comment:
_we will augment some of this information with some considerations that may be specific to the cellxgene curation process_ - isn't this document intended to be **specific** to the curation process?

Reply:
amended this wording